docs/src/dataDisplay/tag/tag.jsx

```diff
@@ -0,0 +1,282 @@
+import {
+    Header,
+    Tag,
+    Typography,
+} from 'react-cm-ui';
+import { camelCase } from 'lodash';
+import PropTypes from 'prop-types';
+import React, { useCallback } from 'react';
+// import {
+//     LABEL_SAMPLE,
+//     COLOR_SAMPLE,
+//     FLUID_SAMPLE,
+//     INVERSE_SAMPLE,
+//     ON_CLICK_SAMPLE,
+//     ON_CLEAR_CLICK_SAMPLE,
+// } from './examples/examples';
+import Block from '../../global/block';
+import ComponentVersionIdentifier from '../../global/componentVersionIdentifier';
+import Highlighter from '../../global/highlighter';
+import Main from '../../global/main';
+import MarkdownContainer from '../../global/markdownContainer';
+/* eslint-disable import/no-named-default, import/extensions */
+import { default as rootDoc } from '!!@advclb/react-docgen-loader!react-cm-ui/dataDisplay/tag/tag';
+/* eslint-enable import/no-named-default, import/extensions */
+
+const propTypes = {
+    location: PropTypes.shape({
+        pathname: PropTypes.string,
+    }).isRequired,
+};
+
+export const LABEL_SAMPLE = `import { Tag } from 'react-cm-ui';
+import React from 'react';
+
+export default class InputSample extends React.Component {
+
+    render() {
+        return (
+            <Tag>Label</Tag>
+        );
+    }
+
+}`;
```

Comment:
We should move all of these to other files, right?

Reply:
Fixed.

Reply:
Resolving.